e2e/src/docker_logs.rs

```diff
@@ -0,0 +1,52 @@
+use anyhow::{anyhow, Context, Result};
+use std::process::Command;
+
+fn find_stablex_container_in_output(output: &str) -> Option<&str> {
+    output
+        .split('\n')
+        .skip(2)
+        .filter_map(|line| line.split(' ').next())
+        .find(|container_name| container_name.starts_with("dex-services_stablex_1"))
+}
+
+fn find_stablex_container() -> Result<String> {
+    let output = Command::new("docker-compose")
```

Comment:
I would rather use something like `docker ps --quiet --filter '^dex-services_stablex_1'` to just get the ID directly, then we don't rely on the output being formatted in a specific way

Reply:
Good suggestion. Didn't know about `--quiet`.